test/ComposerRequireCheckerTest/ASTLocator/LocateASTFromFilesTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace ComposerRequireCheckerTest\ASTLocator;
+
+use ArrayObject;
+use ComposerRequireChecker\ASTLocator\LocateASTFromFiles;
+use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
+use PhpParser\Lexer;
+use PhpParser\Parser\Php7;
+use PHPUnit\Framework\TestCase;
+
+class LocateASTFromFilesTest extends TestCase
```

Comment:
Please add `/** @covers \ComposerRequireChecker\ASTLocator\LocateASTFromFiles */` to prevent coverage leaks